BEGINNERS.md

```diff
@@ -54,9 +54,11 @@ follow the [Vagrant setup instructions](SETUP_VAGRANT.md).
 **This is the best and easiest option for all systems including
 GNU/Linux, BSD, Mac OS X, and Windows.**
 
-As an alternative option, we also have instructions for local installation.
-[Debian-based instructions](SETUP_DEBIAN.md) work for Debian or Ubuntu or
-related, and our general [GUIDE](GUIDE.md) has some notes for other systems.
+As an alternative option, we also have instructions for local
```

Comment:
Sure. I'll have to take a look at this later.

Peter Harpending

On Fri, May 22, 2015 at 05:59:44PM -0700, Aaron Wolf wrote:
> > @@ -54,9 +54,11 @@ follow the [Vagrant setup instructions](SETUP_VAGRANT.md).
> >  **This is the best and easiest option for all systems including
> >  GNU/Linux, BSD, Mac OS X, and Windows.**
> >  
> > -As an alternative option, we also have instructions for local installation.
> > -[Debian-based instructions](SETUP_DEBIAN.md) work for Debian or Ubuntu or
> > -related, and our general [GUIDE](GUIDE.md) has some notes for other systems.
> > +As an alternative option, we also have instructions for local
> 
> Sorry to be anal, but would you be up for redoing it to keep it to the three lines and keep the original first line unchanged?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/snowdriftcoop/snowdrift/pull/310/files#r30940192